notes-webapp/src/main/webapp/vue-app/notes/components/NotesOverview.vue

```diff
@@ -506,6 +506,11 @@ export default {
     }
   },
   methods: {
+    getHomeTitle(title) {
+      const label = `notes.label.note${title}`;
+      const translation = this.$t(label);
+      return translation === label && title || translation;
```

Comment:
```suggestion
      return title === 'Home' && this.$t('notes.label.noteHome') || title;
```
May be just test when `title === 'Home'` to see if we have to apply `this.$t` on it. This will avoid to attempt internationalizing other labels that can match an i18N key